src/app/static/src/app/store/stepper/stepper.ts

```diff
@@ -62,3 +69,12 @@ export const stepper = (existingState: Partial<RootState> | null): Module<Steppe
         mutations
     };
 };
+
+export const dataExplorationStepper = (existingState: Partial<DataExplorationState> | null): Module<StepperState, DataExplorationState> => {
+    return {
+        namespaced,
```

Comment:
Also maybe just a comment here highlighting that getters are omitted and would not be valid for this version of the module